src/Pages/Login/Login.scss

```
@@ -0,0 +1,116 @@
+@import "../../Styles/common.scss";
+
+.Login {
+  a {
+    text-decoration: none;
+    color: #000;
+  }
+  box-sizing: border-box;
+  display: flex;
+  height: 100vh;
```

순서를 아래와 같이 바꾸어주시면 더 코드를 보기에 편할 것 같아요! :) 

```suggestion
  box-sizing: border-box;
  display: flex;
  height: 100vh;
  
    a {
      text-decoration: none;
      color: #000;
    }
```

---

수정완료!